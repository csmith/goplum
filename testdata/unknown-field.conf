check debug.random "check" {
    interval = 2s
    alerts = "foo"
    bazinga = true
}
