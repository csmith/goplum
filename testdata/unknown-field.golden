"error configuring check check: error decoding check settings: unknown configuration key: bazinga"